app/src/main/java/net/programmierecke/radiodroid2/alarm/AlarmReceiver.java

```diff
@@ -215,6 +217,83 @@ protected void onPostExecute(String result) {
         }.execute();
     }
 
+    private void setAlarmVolume(float volume) {
+        try {
+            if (BuildConfig.DEBUG) { Log.d(TAG, "Setting volume to "+volume); }
+            itsPlayerService.SetVolume(volume);
+        } catch (Exception e) {
+            Log.e(TAG, "Couldn't set volume "+e);
```

Comment:
If there is a failure here it means that service is dead or some gone terribly wrong. We should play system sound in such cases. This could be done as a separate PR with improvements I asked in #600